### DISCUSSION TITLE
Consistently different results when running examples in colab vs locally

### DISCUSSION THREAD

**Discussion by techybolek** [microsoft/autogen#467] [Q&A]
I keep getting different results when running AutoGen examples in colab vs locally. For example:

https://github.com/microsoft/autogen/blob/main/notebook/agentchat_groupchat.ipynb

When run in colab I get:
```
 Title: Skill-Mix: a Flexible and Expandable Family of Evaluations for AI models . 
 Published: 2023-10-26T16:55:05Z 

```
When running locally I get:
```
Can LLMs like GPT-4 outperform traditional AI tools in dementia
  diagnosis? Maybe, but not today
Published:  2023-06-02T12:47:45Z

```

**Comment by gagb** [comment]
This could be because of stochasticity in LLM responses. You could try reducing the temperature in llmconfig

**Reply by techybolek** [reply]
The way I understand, stochasticity would express itself by producing different results with each execution. But again, this is not the case. Both environment produces consistent results, they are just different across each environment.
Anyway I did explicitly set the temp to zero and, and, unsurprisingly, this did not change a thing.

**Comment by sonichi** [comment]
It could be due to the environment difference, such as the preinstalled packages.
